src/pages/docs/multiple-forms.mdx

```diff
@@ -35,11 +40,11 @@ The code for the above example looks as follows:
 
 ```
 <div>
-  <JsonForms schema={personSchema}
+  <JsonFormsDispatch schema={personSchema}
```

Comment:
Same here, `JsonFormsDispatch` needs to be used when using Redux.

Reply:
Done, reverted the change.